system-reinstall-bootc/src/prompt.rs

```diff
@@ -1,16 +1,16 @@
-use crate::{
-    prompt,
-    users::{get_all_users_keys, UserKeys},
-};
+use crate::{prompt, users::get_all_users_keys};
 use anyhow::{ensure, Context, Result};
 
+use std::fs::File;
+use std::io::Write;
+
 const NO_SSH_PROMPT: &str = "None of the users on this system found have authorized SSH keys, \
     if your image doesn't use cloud-init or other means to set up users, \
     you may not be able to log in after reinstalling. Do you want to continue?";
 
 fn prompt_single_user(user: &crate::users::UserKeys) -> Result<Vec<&crate::users::UserKeys>> {
     let prompt = format!(
-        "Found only one user ({}) with {} SSH authorized keys. Would you like to import it and its keys to the system?",
+        "Found only one user ({}) with {} SSH authorized keys. Would you like to import it's SSH authorized keys into the root user on the new bootc system?",
```

Comment:
Minor nit, "its" here is correct (not "it's") as it is a possessive. While we're here would it also look nicer to line wrap?